po4a-update-templates.sh

```diff
@@ -77,16 +77,16 @@ do
             --master "$file" \
             --master-charset "UTF-8" \
             --po "$po_file" ; then
-        echo ""
-        echo "Error updating $lang PO file for: $adoc_file"
+        echo ''
+        echo 'Error updating $lang PO file for: $adoc_file'
```

Comment:
Not much of an error as it doesn't cause termination of the script.  Should it?

Reply:
Well, it continues to update the rest of the po files. Depends on whether you want it to just exit regardless or not.

I'm working on the quote drama now. Getting rid of them all except where needed.